pkg/envoy/sort_test.go

```diff
@@ -20,45 +20,55 @@ import (
 	"github.com/cilium/cilium/pkg/checker"
 
 	"github.com/cilium/proxy/go/cilium/api"
-	envoy_api_v2_core "github.com/cilium/proxy/go/envoy/api/v2/core"
-	envoy_api_v2_route "github.com/cilium/proxy/go/envoy/api/v2/route"
+	envoy_config_core "github.com/cilium/proxy/go/envoy/config/core/v3"
+	envoy_config_route "github.com/cilium/proxy/go/envoy/config/route/v3"
+	envoy_type_matcher "github.com/cilium/proxy/go/envoy/type/matcher/v3"
+
 	. "gopkg.in/check.v1"
 )
 
 type SortSuite struct{}
 
 var _ = Suite(&SortSuite{})
 
-var HeaderMatcher1 = &envoy_api_v2_route.HeaderMatcher{
-	Name:                 "aaa",
-	HeaderMatchSpecifier: &envoy_api_v2_route.HeaderMatcher_RegexMatch{RegexMatch: "aaa"},
+var HeaderMatcher1 = &envoy_config_route.HeaderMatcher{
+	Name: "aaa",
+	HeaderMatchSpecifier: &envoy_config_route.HeaderMatcher_SafeRegexMatch{SafeRegexMatch: &envoy_type_matcher.RegexMatcher{
+		EngineType: googleRe2, Regex: "aaa"},
+	},
```

Comment:
nit: alignment
Please check changes below as well.

Reply:
Alignment is OK as long as it passes `go vet` which is run by `make`.